apache/arrow-ballista
# [Python] Fix Ballista Python bindings

# Which issue does this PR close?



N/A

 # Rationale for this change


We can now run queries in Python again!

```
>>> import ballista
>>> ctx = ballista.BallistaContext("localhost", 50050)
>>> df = ctx.sql("SELECT 1")
>>> df.show()
+----------+
| Int64(1) |
+----------+
| 1        |
+----------+
```

# What changes are included in this PR?



- Copy latest from `arrow-datafusion-python` 
- Change names from datafusion to ballista
- Add BallistaContext from https://github.com/apache/arrow-ballista/pull/58
- Update CI to no longer use an old nightly Rust

# Are there any user-facing changes?


No
